.flake8

```diff
@@ -1,36 +1,44 @@
 [flake8]
 min_python_version = 3.7.0
-max-line-length = 88
-ban-relative-imports = true
-# flake8-use-fstring: https://github.com/MichaelKim0407/flake8-use-fstring#--percent-greedy-and---format-greedy
-format-greedy = 1
-inline-quotes = double
-enable-extensions = TC, TC1
-type-checking-exempt-modules = typing, typing-extensions
-eradicate-whitelist-extend = ^-.*;
+enable-select =
```

Comment:
I cannot find this option :thinking: Do you mean [extend-select](https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-extend-select)? Or `enable-extensions` as described in https://pypi.org/project/flake8-type-checking/?

Reply:
I do mean `extend-select` -- looks like this silently breaks typechecking messages :laughing: